### ISSUE TITLE
pk6 > pk7 changes

### ISSUE THREAD

**Issue by Kirzi** [porygonco/porybox#616]
So I noticed a few things when I was looking into Bank's pk6 > pk7 conversion and did some science.

What I did:

1. Caught a Magikarp in SS, then transferred it to White > AS.  (I needed to start with something in gen 4 to check the gen 4 encounter flag)
2. Gave it some markings
3. Did some Amie stuff to get some fullness and affection (not actually relevant, just leaving it here for the sake of explaining changes)
4. Maxed its EVs using vitamins to unlock secret training, then did one of the stages
5. Used an EV reducing berry, then punched a bag a little to get some punches in
6. Dumped AS save, transferred to Sun, dumped Sun save. Extracted the pk6/pk7.
7. Renamed pk7 to pk6 to upload to Porybox

Changes in Porybox display:

- Markings are gone
- Encounter type (gen 4) wiped; now is just empty
- Super training box now says "Secret Training enabled? No" instead of "Yes"
- Fullness/affection wiped (disregard; this gets reset on trade anyway so it's not specific to a transfer)

In other words, aside from the marking changes, we'll need to remove the "encounter type" line for gen 4 Pokemon and remove the secret training enabled line for pk7s. 

pk6: https://www.porybox.com/#/pokemon/6c56666633fdf0e653a1b2abc8e657e4
pk7: https://www.porybox.com/#/pokemon/26671d86ca917e2ab38c1f2c36496918
